src/prefect/server/database/orm_models.py

```diff
@@ -878,7 +878,18 @@ def job_variables(self):
     )
 
     concurrency_limit: Mapped[Union[int, None]] = mapped_column(
-        sa.Integer, default=None, nullable=True
+        sa.Integer,
+        server_default=None,
```

Comment:
did this server default get added w/o a db migration? 

or well - why did this get added?

Reply:
Oh. TIL [alembic doesn't pick up server_default changes by default](https://github.com/sqlalchemy/alembic/discussions/1204) so I guess this doesn't matter but I think this line isn't doing anything as the DDL has already been generated and the updated DDL to alter the column w/ a db-server-side default isn't tracked via migrations.